dev-utils/storybook/stories/lit-components/themeConfiguration.js

```diff
@@ -0,0 +1,34 @@
+import { html } from "lit-element";
+
+export function themeConfiguration() {
+    return html`
+        <style>
+            @font-face {
+                font-family: "Jost Light";
+                src: url("public/fonts/jost_light.ttf");
```

Comment:
Did you check this path is correct?

Reply:
No, I completely missed this part, thank you!
Although, now that I think about it, I'm not sure where it's supposed to point to. I don't see any fonts in `poly-look`.

Reply:
No, they are in the common `assets` directory.